### PR TITLE
Use the new `Debugger.getEventHandlerCounts` method

### PR DESCRIPTION
**Pull request by jcmorrow**
OK, the protocol version which includes [Debugger.getEventHandlerCounts](https://static.replay.io/protocol/tot/Debugger/#method-getEventHandlerCounts) has been released, so this is ready for review!

### CODE REVIEW

**Comment by jcmorrow** [comment]
Requires https://github.com/replayio/backend/pull/6432

**Comment by jcmorrow** [comment]
I tried following the docs to reconstruct playwright fixtures, but all of my playwright tests just fail anyways, so I don't know how to fix that.

File: packages/shared/client/ReplayClient.ts [hunk 1]
```diff
@@ -302,19 +302,12 @@ export class ReplayClient implements ReplayClientInterface {
...
-  }
-
-  async getEventCountForType(eventType: EventHandlerType): Promise<number> {
     const sessionId = this.getSessionIdThrows();
-    const { count } = await client.Debugger.getEventHandlerCount({ eventType }, sessionId);
-    return count;
+    const { counts }: { counts: { type: string; count: number }[] } =
+      await client.Debugger.getEventHandlerCounts({ eventTypes }, sessionId);
+    const countsObject = Object.fromEntries(counts.map(({ type, count }) => [type, count]));
+    console.log({ counts, countsObject });
```
Comment (bvaughn) [inline packages/shared/client/ReplayClient.ts:309]:
```suggestion
```

**Review by bvaughn — APPROVED** [review]
Looks good 😄

**Comment by jcmorrow** [comment]
Did not get around to merging this today, but assuming that CI passes I'll try and get it in tomorrow!

**Comment by jcmorrow** [comment]
@bvaughn I have tried to fix the playwright tests, but they all just fail for me locally no matter what I do. Would you mind figuring out how to get those to pass with the new protocol endpoint?

**Comment by markerikson** [comment]
@jcmorrow I can pick this up

**Comment by jcmorrow** [comment]
Thank you!!

**Comment by markerikson** [comment]
@jcmorrow oh drat, it's the component screenshot tests, not the new E2E tests, and I haven't managed to get my system set up to recreate those either yet

**Comment by bvaughn** [comment]
> @bvaughn I have tried to fix the playwright tests, but they all just fail for me locally no matter what I do. Would you mind figuring out how to get those to pass with the new protocol endpoint?

@jcmorrow Have you tried the steps in the [README](https://github.com/replayio/devtools/blob/main/packages/bvaughn-architecture-demo/playwright/README.md)? Which one fails?

You'll need to [update the fixture data](https://github.com/replayio/devtools/blob/main/packages/bvaughn-architecture-demo/playwright/README.md#updating-protocol-fixture-data) since you've added a new protocol API endpoint in this PR. That should be it though, I think!

**Comment by jcmorrow** [comment]
I just realized this never got merged. Sorry, I thought I was handing it off.

**Comment by markerikson** [comment]
Lost in the shuffle, but also we're still pretty preoccupied with landing the Points refactor PR atm, to the exclusion of everything else.

**Comment by jcmorrow** [comment]
OK, I'll continue to leave it to y'all then.

**Pull request merged**